### PR TITLE
ipq40xx: Support hard button for WPJ419

### PR DESCRIPTION
**Pull request by phi-nguyendp**
Compex WPJ419 has a button on a GPIO pin.

Signed-off-by: Phi Nguyen <phind.uet@gmail.com>

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
I don't find any reference to a reset button/GPIO, just did a quick search, though.
Can you provide any documentation on that?

**Comment by ita93** [comment]
Hi, 
https://compex.com.sg/wp-content/uploads/2020/06/wpj419-7a04-v2-sl-194-1.pdf
As described in this document (page 2), there is a Reset button

**Comment by adrianschmutzler** [comment]
Ah, fine thanks. Some minor formal stuff:
In your commit title, I'd refer to a "reset" button; and please use lower case after prefix:
ipq40xx: support reset button for WPJ419

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wpj419.dts [hunk 1]
```diff
@@ -348,6 +348,15 @@
 			status = "okay";
 		};
 	};
+	
+	gpio-keys {
```
Comment (adrianschmutzler) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wpj419.dts:352]: name should be "keys"

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wpj419.dts [hunk 2]
```diff
@@ -348,6 +348,15 @@
...
+		compatible = "gpio-keys";
```
Comment (adrianschmutzler) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wpj419.dts:353]: add empty line after compatible.

**Comment by ita93** [comment]
Hi, I have updated it.

**Comment by adrianschmutzler** [comment]
Thanks, will be merged during the day.

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wpj419.dts [hunk 3]
```diff
@@ -348,6 +348,16 @@
 			status = "okay";
 		};
 	};
+	
+	keys {
+		compatible = "gpio-keys";
+
```
Comment (adrianschmutzler) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wpj419.dts:354]: This "empty" line should really be empty and not contain any tabs or other whitespaces. I've removed that myself, so no need to do anything now. Just to tell you for the next time you send a patch. :-)

**Pull request closed without merging**